libs/base/System/File.idr

```diff
@@ -288,26 +307,61 @@ fPoll (FHandle f)
     = do p <- primIO (prim__fPoll f)
          pure (p > 0)
 
+||| Produce enough @Fuel@ to read the given number
+||| of lines.
 export
-readFile : HasIO io => String -> io (Either FileError String)
-readFile file
+lineCount : Nat -> Fuel
```

Comment:
I think we can reuse `Data.Fuel`'s `limit` instead of introducing this identical definition.

Reply:
Doh! Yeah, that'd be the way to go.